cpp/src/io/comp/nvcomp_adapter.cpp

```diff
@@ -447,6 +454,7 @@ std::optional<std::string> is_decompression_disabled(compression_type compressio
 size_t required_alignment(compression_type compression)
 {
   switch (compression) {
+    case compression_type::GZIP:
```

Comment:
returning the same value as deflate because it's the same compression format; the only difference is an additional header in GZIP.